src/wayland/seat/keyboard/keymap_file.rs

```diff
@@ -63,12 +64,14 @@ struct SealedFile {
 
 impl SealedFile {
     fn new(keymap: &CStr) -> Result<Self, std::io::Error> {
-        let name = CString::new("smithay-keymap").expect("File name should not contain interior nul byte");
+        /* Previously we use memfd_create()  
```

Comment:
Android actually supports memfd_create, you'll need to upgrade to nix 0.25 to get it.

Reply:
Aaah okayy!!